playground/TestShop/CatalogService/Program.cs

```diff
@@ -8,19 +8,10 @@
 
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddProblemDetails();
-builder.Services.AddSwaggerGen();
```

Comment:
You can also remove the `builder.Services.AddEndpointsApiExplorer();` alongside this.